instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/patches/active_job_callbacks.rb

```diff
@@ -72,6 +73,10 @@ def job_attributes(job)
               'messaging.destination' => job.queue_name,
               'messaging.message_id' => job.job_id,
               'messaging.active_job.provider_job_id' => job.provider_job_id,
+              'messaging.active_job.enqueued_at' => job.enqueued_at,
+              'messaging.active_job.enqueued_duration_ms' => milliseconds_since(job.enqueued_at),
+              'messaging.active_job.first_enqueued_at' => job.first_enqueued_at,
+              'messaging.active_job.enqueued_and_retry_duration_ms' => milliseconds_since(job.first_enqueued_at),
```

Comment:
The time since the job was first enqueued.  This captures:
* Time spent enqueued
* Time spent waiting to be retried
* Duration of prior executions